test/controllers/auth/GetPersonDetailsActionSpec.scala

```diff
@@ -60,6 +60,7 @@ class GetPersonDetailsActionSpec extends BaseSpec {
       ConfidenceLevel.L50,
       None,
       None,
+      null,
```

Comment:
What argument is this? Surprised you can have a null here

Reply:
Replaced null with Set() - empty set , thanks